robots/commenter/main.go

```diff
@@ -70,6 +70,8 @@ func flagOptions() options {
 	flag.StringVar(&o.comment, "comment", "", "Append the following comment to matching issues")
 	flag.BoolVar(&o.useTemplate, "template", false, templateHelp)
 	flag.IntVar(&o.ceiling, "ceiling", 3, "Maximum number of issues to modify, 0 for infinite")
+	flag.IntVar(&o.spamCeiling, "spam-ceiling", 3, "Maximum number of the same comment in sequence to post, 0 for infinite")
```

Comment:
if anything, this needs to be a neutral and descriptive `comment-ceiling`. Considering it "spam" is a subjective view for your current situation and when I look at this setting, it is unclear what exactly "spam" means.